test/builder/builder.go

```diff
@@ -641,7 +641,7 @@ const describeTemplate = `{{decorate "bold" "Name"}}:	{{ .PipelineName }}
  NAME	TYPE	DESCRIPTION	DEFAULT VALUE
 {{- range $i, $p := .Pipeline.Spec.Params }}
 {{- if not $p.Default }}
- {{decorate "bullet" $p.Name }}	{{ $p.Type }}	{{ $p.Description }}	{{ "" }}
```

Comment:
Is not the same change required in cmd also then

Reply:
If I have understood that right, it's already there. Can you please check once [here](https://github.com/tektoncd/cli/blob/master/pkg/cmd/pipeline/describe.go#L63)

Reply:
Ok, so this was missed here. LGTM